basics/src/test/java/framework/ReadProperties.java

```diff
@@ -16,4 +16,15 @@ public String ReadBrowserType() throws IOException {
 		
 		return properties.getProperty("BrowserType");
 	}
+	
+	public String ReadCSVName() throws IOException {
+		Properties properties = new Properties();
+		String propFileName = "config.properties";
+		InputStream inputStream = getClass().getClassLoader().getResourceAsStream(propFileName);
+		
+		properties.load(inputStream);
+		inputStream.close();
```

Comment:
Should wrap this in a try/catch/finally. Close should go in the finally section.